src/OSPSuite.Core/Chart/ChartSettings.cs

```diff
@@ -14,6 +14,14 @@ public enum LegendPositions
       BottomInside
    }
 
+   public enum GridGroupRowFormats
```

Comment:
Move out of ChartSettings. This is a concept that is relevant for all UI using a GridView

Reply:
@msevestre this is not ready yet